eslint-example/main.js

```diff
@@ -0,0 +1,13 @@
+/* global getUsefulContents */
+// not supported in Firefox
+// import { getUsefulContents } from "file";
```

Comment:
I don't think it's so helpful to put this comment in here. The linter will complain if someone accidentally adds an import statement. The `es6: true` flag implies that `import` is not supported.